recipes/hunspell/all/CMakeLists.txt

```
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
 
 project(hunspell VERSION "${CONAN_hunspell_VERSION}" LANGUAGES CXX)
 
-include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-conan_basic_setup(TARGETS KEEP_RPATHS)
-set(src src/hunspell)
+set(src "${HUNSPELL_SRC_DIR}/src/hunspell")
```

```suggestion
set(src "${CMAKE_CURRENT_SRC_DIR}/src/hunspell")
```
I don't think you need to pass it via recipe. The cmake source folder should be the same path.

---

@uilianries 
Unfortunately this does not seem to work, `CMAKE_CURRENT_SRC_DIR` is not set at that point
```
message(FATAL_ERROR "CMAKE_CURRENT_SRC_DIR: '${CMAKE_CURRENT_SRC_DIR}'")
```
Results in 
```
CMAKE_CURRENT_SRC_DIR: ''
```

I will dig a bit to see if there is not another variable that contains the correct value.

---

I see it must be `CMAKE_CURRENT_SOURCE_DIR` instead, trying that now